src/sanitizer.js

```diff
@@ -345,6 +347,25 @@ function uponSanitizeAttribute(node, data) {
   data.attrValue = attrValue;
 }
 
+/**
+ * @param {!Node} node
+ * @this {{removed: !Array}} Contains list of removed elements/attrs so far.
+ */
+function afterSanitizeAttributes(node) {
+  filterSplice(this.removed, r => {
+    if (r.from === node && r.attribute) {
+      const {name, value} = r.attribute;
+      // Restore the `on` attribute which DOMPurify incorrectly flags as an
+      // unknown protocol due to presence of the `:` character.
+      if (name.toLocaleLowerCase() === 'on') {
```

Comment:
why do we do toLocaleLowerCase? toLowerCase should suffice? Similarly in this line, https://github.com/ampproject/amphtml/blob/f52486bbf853e2907909c12b1c90e2f917ec2bf7/src/sanitizer.js#L300

Reply:
This was to be consistent with DOMPurify's internal usage, but `toLowerCase` should be fine. Fixed.

Reply:
gotcha. Thanks